fuel/src/main/kotlin/com/github/kittinunf/fuel/core/requests/RepeatableBody.kt

```diff
@@ -0,0 +1,72 @@
+package com.github.kittinunf.fuel.core.requests
+
+import com.github.kittinunf.fuel.core.Body
+import java.io.ByteArrayInputStream
+import java.io.InputStream
+import java.io.OutputStream
+
+/**
+ * A Repeatable Body wraps a body and on the first [writeTo] it keeps the bytes in memory so it can be written again.
+ *
+ * Delegation is not possible because the [body] is re-assigned, and the delegation would point to the initial
+ *   assignment.
+ */
+data class RepeatableBody(
+    var body: Body
+) : Body {
```

Comment:
It says it right there in the comment why we can't:

```kotlin
/*
 * Delegation is not possible because the [body] is re-assigned, and the delegation would point to the initial
 *   assignment.
 */
```

Reply:
Ahhhh sorry!! 🙏